numpy/_core/meson.build

```diff
@@ -506,6 +506,10 @@ if cc.has_function_attribute('visibility:hidden') and host_machine.system() != '
 endif
 cdata.set('NPY_VISIBILITY_HIDDEN', visibility_hidden)
 
+# if set, we're using lapack_lite
+if have_blas
+  cdata.set10('HAVE_BLAS', have_blas)
+endif
```

Comment:
Pretty sure we can have blas, but not lapack, so I think this should use `have_lapack`.
(Not sure this is actually plausible/possible in practice, but seems more specific anyway.)

Reply:
It's very rare, but possible indeed. I think older versions of Arch Linux shipping a BLAS-only OpenBLAS build is the one case that I saw more than once - but that is gone now.

Reply:
Maybe a little more descriptive would be nice: `HAVE_EXTERNAL_LAPACK`.

Reply:
Thanks for pointing this out, I conflated with the `allow-noblas` build option.